src/bidiTab/bidiTab.ts

```diff
@@ -107,12 +107,7 @@ function createCdpConnection() {
     }
   }
 
-  return new CdpConnection(
-    new WindowCdpTransport(),
-    (...messages: unknown[]) => {
-      log(LogType.cdp, ...messages);
```

Comment:
The first argument is used to define a container on the Mapper Debug page, where to put the log. This change split CDP and BiDi send and receive messages into different containers.